components/permalink_view/permalink_view.test.tsx

```diff
@@ -304,7 +306,6 @@ describe('components/PermalinkView', () => {
                         ...initialState.entities,
                         general: {
                             config: {
-                                FeatureFlagCollapsedThreads: 'true',
```

Comment:
this looks like the only change, we should avoid changing the rest of the file unless needed.

Reply:
These are actually auto lint changes done by `make fix-style` command.